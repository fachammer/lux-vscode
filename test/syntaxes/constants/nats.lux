123
123,456,789,000
1,,2,,3,,,4,,,5,,,6
