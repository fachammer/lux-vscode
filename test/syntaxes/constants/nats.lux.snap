>123
#^^^ source.lux constant.numeric.nat.lux
>123,456,789,000
#^^^^^^^^^^^^^^^ source.lux constant.numeric.nat.lux
>1,,2,,3,,,4,,,5,,,6
#^^^^^^^^^^^^^^^^^^^ source.lux constant.numeric.nat.lux
>